mitiq/interface/mitiq_braket/conversions.py

```diff
@@ -122,6 +123,11 @@ def _translate_cirq_operation_to_braket_instruction(
     Raises:
         ValueError: If the operation cannot be converted to Braket.
     """
+    # Measurement gates do not exist in Braket.
+    if isinstance(op.gate, cirq_ops.MeasurementGate):
+        warn("Measurement gate removed when converting from Cirq to Braket.")
+        return []
```

Comment:
Confirming my recollection that braket doesn't have measurements. I wonder why. Do they just assume everything gets measured at the end?

Anyway, great to have this warning as well.

Reply:
AFAIK, Braket assumes all qubits are measured.